### ISSUE TITLE
WCAG 2 AAA Guideline 4.1.1 Failure 

### ISSUE THREAD

**Issue by TharenaMelishka** [FortAwesome/Font-Awesome#16426]
**Describe the bug**
I ran an accessibility report through Koa11y [https://open-indy.github.io/Koa11y/](https://open-indy.github.io/Koa11y/) , and my single line of font-awesome code generated the following failure:

WCAG2AAA.Principle4.Guideline4_1.4_1_1.F77
error: Duplicate id attribute value "font-awesome-5-kit-css" found on the web page.

<link href="https://kit-free.fontawesome.com/releases/latest/css/free-v4-font-face.min.css" media="all" rel="stylesheet" id="font-awesome-5-kit-css">

<link href="https://kit-free.fontawesome.com/releases/latest/css/free.min.css" media="all" rel="stylesheet" id="font-awesome-5-kit-css">

#font-awesome-5-kit-css

**To Reproduce**
As I don't want to reveal my company's fontawesome code line, I'm not sure how to codepen this.  However:  This should be reproduce-able by using Koa11y's tool at AAA accessibility requirements to reveal that two separate lines of fontawesome code maintain the same ID.  A review of the straight fontawesome code should reveal that these two lines of code have the same ID which is against the accessibility guidelines.

**Expected behaviour**
For anyone in Ontario, Canada, meeting accessibility compliance before 2025 at the latest (depending on the size and type of business) is vital for businesses large and small.  A triple-A badge is the highest standard and difficult to achieve.  It would be a great boon to keeping Font-awesome accessible to e-readers and other assistive technology to work within these accessibility guidelines in what should be tiny but meaningful way, such as adding a version number to the IDs of these two pieces lines of code.  If this was simply a "warning", it would be less of a concern.  Each individual line of css code should have a different id to meet WCAG requirements.   This single outright failure would cause a website to lose AAA accreditation, or necessitate a different solution to font-awesome to maintain accessibility standards. 

Please review:
[https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-parses.html](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-parses.html)
[https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/H93](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/H93)
[https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/H94](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/H94)

While there are other measures around using font-awesome icons that need to be implemented in order for them to be fully accessible, this is the only one I have come across that causes a full failure and not simply a warning.

**Version and implementation**
Version:  font awesome free 4/5
Browser and version: browser irrelevant (within the code itself)

kit-free.fontawesome.com/releases/latest/css/free-v4-font-face.min.css
kit-free.fontawesome.com/releases/latest/css/free.min.css

both have the same ID:
id="font-awesome-5-kit-css"

**Bug report checklist**
- [x] I have filled out as much of the above information as I can
- [ ] I have included a test case because my odds go _way_ up that the team can fix this when I do -- not possible in this instance, must be reproduced independently.
- [x] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate

Many thanks!

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

This should have been fixed in #16149

Please save your kit again as described here: https://github.com/FortAwesome/Font-Awesome/issues/16149#issuecomment-594572931

**Comment by TharenaMelishka** [comment]
Bless you it did, thank you !  <3
